go/vt/vtctl/workflow/materializer_test.go

```diff
@@ -2427,7 +2431,7 @@ func TestKeyRangesEqualOptimization(t *testing.T) {
 				if tablet.Keyspace != targetKs || tablet.Type != topodatapb.TabletType_PRIMARY {
 					continue
 				}
-				env.tmc.expectVRQuery(int(tablet.Alias.Uid), mzSelectFrozenQuery, &sqltypes.Result{})
+				//env.tmc.expectVRQuery(int(tablet.Alias.Uid), mzSelectFrozenQuery, &sqltypes.Result{})
```

Comment:
Why is this commented out?

Reply:
Removed. We don't need it since we have removed the `frozen` check.